packages/workers-shared/router-worker/src/index.ts

```diff
@@ -16,7 +29,9 @@ interface Env {
 export default {
 	async fetch(request: Request, env: Env, ctx: ExecutionContext) {
 		let sentry: ReturnType<typeof setupSentry> | undefined;
-		const maybeSecondRequest = request.clone();
+		const analytics = new Analytics();
```

Comment:
NIT: similar to the PerformanceTimer, why not pass in the env.ANALYTICS and env.ENVIRONMENT here in the constructor, rather than in the write method?